tests/helpers/Participant.ts

```diff
@@ -135,7 +151,7 @@ export class Participant {
         if (!options.skipDisplayName) {
             // @ts-ignore
             config.userInfo = {
-                displayName: options.displayName || this._name
+                displayName: this._displayName = options.displayName || this._name
```

Comment:
Did you intend this to be an assignment? Since there is a getter, shouldn't this be `this.displayName` ?

Reply:
Yep, this is where we set it from the options... But I can drop part of the later changes where I use a constant instead of call the dispalyName getter.

Reply:
No strong opinion, you pick!